compiler/rustc_transmute/src/layout/tree.rs

```diff
@@ -268,6 +366,10 @@ pub(crate) mod rustc {
                     }))
                 }
 
+                ty::Char => Ok(Self::char(cx.tcx().data_layout.endian.into())),
+
+                ty::Alias(_kind, ty) => Self::from_ty(ty.to_ty(cx.tcx()), cx),
```

Comment:
(Copying [this comment](https://github.com/rust-lang/rust/commit/218364989b5940ecb71ec869a18124afed43b1ad#r155776593) from @jswrenn):

> Can you write a test for a type alias of a type alias?

Reply:
Done